pkg/destroy/gcp/compute.go

```diff
@@ -111,6 +111,29 @@ func (o *ClusterUninstaller) deleteInstanceGroup(ig nameAndZone) error {
 	return nil
 }
 
+// destroyInstanceGroups searches for instance groups across all zones that have a name that starts with
```

Comment:
I think we also need to make sure health-checks (ILB), http-health-checks (NLB) are destroyed.

Reply:
> I think we also need to make sure health-checks (ILB), http-health-checks (NLB) are destroyed.

I think they are covered here: https://github.com/openshift/installer/pull/2448/files/39c63115aaf52181323d8b76807c3469465969cc#diff-4781d154886c4da10a007bb219f27d8aR140-L140

I have noticed any leaked resources in my personal install/deletes.